static/js/util/profile_edit.js

```diff
@@ -312,22 +312,22 @@ export function boundMonthYearField(keySet, label) {
   // fullWidth means set width = 100% instead of 256px
   return <div>
     <label>{label}</label> <TextField
-    floatingLabelText="MM"
-    style={{
-      maxWidth: "2em"
-    }}
-    fullWidth={true}
-    value={monthYear.month !== undefined ? monthYear.month : ""}
-    onChange={e => setNewDate(e.target.value, undefined)}
-  /> <span className="slash">/</span> <TextField
-    floatingLabelText="YYYY"
-    style={{
-      maxWidth: "4em"
-    }}
-    fullWidth={true}
-    value={monthYear.year !== undefined ? monthYear.year : ""}
-    onChange={e => setNewDate(undefined, e.target.value)}
-  />
+      floatingLabelText="MM"
+      style={{
+        maxWidth: "2em"
+      }}
+      fullWidth={true}
+      value={monthYear.month !== undefined ? monthYear.month : ""}
+      onChange={e => setNewDate(e.target.value, undefined)}
+    /> <span className="slash">/</span> <TextField
+      floatingLabelText="YYYY"
+      style={{
+        maxWidth: "4em"
+      }}
+      fullWidth={true}
+      value={monthYear.year !== undefined ? monthYear.year : ""}
+      onChange={e => setNewDate(undefined, e.target.value)}
+    />
```

Comment:
I did the `/> <span` so that React knows there's a space between the two elements. If they're on separate lines React assumes I meant to concatenate them together without any spaces. But it looks like here I can move the spaces inside the `span`